packages/js/product-editor/src/components/attribute-control/attribute-control.tsx

```diff
@@ -157,6 +158,10 @@ export const AttributeControl: React.FC< AttributeControlProps > = ( {
 	};
 
 	const openEditModal = ( attribute: ProductAttribute ) => {
+		recordEvent( 'product_options_edit', {
+			source: TRACKS_SOURCE,
+			attribute: attribute.name,
```

Comment:
Similar to the above, I don't see a benefit in sending the attribute name to tracks.